### ISSUE TITLE
Update translations for v0.14.0 before release

### ISSUE THREAD

**Issue by alex4401** [alex4401/mediawiki-extensions-DataMaps#24]
- [x] Internally promote v0.14.0 in-dev branch to the translation portal
- [x] Notify translators
- Pull changes:
- - [x] French
- - [x] Spanish
- - [x] Thai

**Comment by alex4401** [comment]
Blocked by #29, which can be added in a sub-release if needed.

**Comment by alex4401** [comment]
#29 postponed 'till v0.14.1.
